src/Text/LLVM/PP.hs

```diff
@@ -1353,3 +1368,11 @@ structBraces body = char '{' <+> body <+> char '}'
 
 ppMaybe :: Fmt a -> Fmt (Maybe a)
 ppMaybe  = maybe empty
+
+-- | Throw an error if the ?config version is older than the given version. The
```

Comment:
A minor formatting improvement to make `?config` appear as inline code in the rendered Haddocks:

```suggestion
-- | Throw an error if the @?config@ version is older than the given version. The
```